test/e2e-tests.sh

```diff
@@ -48,18 +48,27 @@ parallelism=""
 use_https=""
 (( MESH )) && parallelism="-parallel 1"
 
-if (( HTTP )); then
+if (( HTTPS )); then
   parallelism="-parallel 1"
   use_https="--https"
+  turn_on_auto_tls
+  kubectl apply -f ./test/config/autotls/certmanager/caissuer/
+  add_trap "kubectl delete -f ./test/config/autotls/certmanager/caissuer/ --ignore-not-found" SIGKILL SIGTERM SIGQUIT
```

Comment:
when a signal happens should we invoke `turn_off_auto_tls` ?

note: don't block this PR